src/statistics/slice_statistics.rs

```diff
@@ -133,10 +135,8 @@ impl<D: AsMut<[f64]> + AsRef<[f64]>> Data<D> {
 }
 
 #[cfg(feature = "rand")]
-impl<D: AsRef<[f64]>> ::rand::distributions::Distribution<f64> for Data<D> {
+impl<D: AsRef<[f64]>> ::rand::distr::Distribution<f64> for Data<D> {
     fn sample<R: ::rand::Rng + ?Sized>(&self, rng: &mut R) -> f64 {
-        use rand::prelude::SliceRandom;
```

Comment:
I think it makes more sense to move the `use rand::prelude::IndexedRandom;` here for consistency with other `impl rand::distr::Distribution<...>` blocks